app/e2e/incomingCall.test.ts

```diff
@@ -27,16 +27,18 @@ describe('Incoming Call', () => {
     await expect(element(by.text('Ahoy!'))).toBeVisible();
     await expect(element(by.text('test_email@twilio.com'))).toBeVisible();
 
-    await twilioClient.calls.create({
+    /**
+     * Wait for 10 seconds to let the registration settle.
+     */
+    await new Promise((resolve) => setTimeout(resolve, 10000));
```

Comment:
always 10s? How did we come up with this value?

Reply:
10s does feel like a long time to let the registration settle. Doesn't registration settle when you verify "Ahoy!" on line 27-28

Reply:
Yeah that's when the RN Promise is resolved by the native layer but I suspect there's some things that are happening on the native layer after that. It seems to help.